ra/registration-authority.go

```diff
@@ -216,10 +224,11 @@ func (ra *RegistrationAuthorityImpl) validateContacts(contacts []*core.AcmeURL)
 			continue
 		case "mailto":
 			rtt, err := validateEmail(contact.Opaque, ra.DNSResolver)
-			ra.stats.TimingDuration("RA.DNS.RTT.MX", rtt, 1.0)
+			ra.stats.TimingDuration("RA.DNS.RTT", rtt, 1.0)
```

Comment:
Because it's no longer doing an MX lookup. But you're right, for consistency this should turn into `.A`. Even better: `DNSResolver` should take a stats object on construction and take care of the stats reporting internally, so fewer methods have to include an rtt parameter.